doc/smart-switch/high-availability/smart-switch-ha-dpu-scope-dpu-driven-setup.md

```diff
@@ -66,26 +66,36 @@ From these characteristics, here is the main differences between `DPU-Scope-DPU-
 
 ## 4. Network Physical Topology
 
-The network physical topology for `DPU-Scope-DPU-Driven` setup will be the same as the ENI-level HA setup, e.g. where the NPU/DPU is placed and wired. The main difference being the HA scope on DPU-level.
+The network physical topology for `DPU-Scope-DPU-Driven` setup will be the same as the ENI-level HA setup, e.g. where the NPU/DPU is placed and wired. The main difference is that the HA scope is on the level of all ENIs or a group of ENIs rather than a single ENI. The current deployment will use only 1 HA scope per DPU, so all ENIs on the DPU are mapped to the same HA scope.
 
 This results in the following differences in the network physical topology and the figure below captures the essence of the differences:
 
 ![](./images/ha-scope-dpu-level.svg)
 
 ### 4.1. DPU-level HA scope
 
-With the HA scoped moved from ENI-level to DPU-level, all ENIs on the active DPU will be treated as active and handles traffic, as the figure shows below:
+In the current deployment with a single HA scope per DPU, Active-Standby deployment, means that the NPUs will direct traffic for all ENIs only to the Active DPU.
 
 ### 4.2. DPU-level NPU to DPU traffic forwarding
 
 With the DPU-level HA scope, the traffic forwarding from NPU to DPU will be done on DPU level:
 
-- Each DPU pair will use a dedicated VIP for traffic forwarding from NPU to DPU.
+- Each DPU pair will use a dedicated VIP per HA scope for traffic forwarding from NPU to DPU.
 - All VIPs for all DPUs in the SmartSwitch will be pre-programmed into the NPU and advertised to the network as a single VIP range (or subnet).
 - NPU will setup a route to match the packet on the destination VIP, instead of ACLs to match both the VIP and inner MAC.
 
 The data path and packet format will follow the same as the ENI-level HA setup [as described in the main HA design doc](./smart-switch-ha-hld.md#421-eni-level-npu-to-dpu-traffic-forwarding).
 
+### 4.3 DPU datapath forwarding capability
+
+Some DPUs can perform Active-Active forwarding in the datapath which is a superset of Active-Standby. The main differences in Active-Active forwarding are
+- Once the flow is setup and synced between the DPUs, both DPUs can forward in case traffic lands on any of them.
+- There is no concept of role-switchover since both are Active. There is only planned shutdown of 1 DPU in which case the other DPU takes over.
```

Comment:
the second item doesn't really hold true, because it is possible to model the flow owner as ENI-level role and then switchover the role. 

imo, it will be better to simply ignore this item and say to start simple, we will use planned shutdown as planned switchover as it is the most straightforward way without introducing any complex state machine transition.

Also the workflow below has already got the ideal of using the planned shutdown as planned switchover covered.

Reply:
removed this section as per discussion on teams

Reply:
thanks a lot! this reduces the confusion a lot and we will work on the A-A forwarding as a separate update.